data/profile.d/fedora-kinoite.conf

```diff
@@ -9,3 +9,11 @@ base_profile = fedora-kde
 # Match os-release values.
 os_id = fedora
 variant_id = kinoite
+
+[Storage]
+default_scheme = BTRFS
+btrfs_compression = zstd:1
+default_partitioning =
+    /
+    /var (btrfs)
```

Comment:
You will have to update the `ProfileConfigurationTestCase.test_fedora_profiles` test and replace `WORKSTATION_PARTITIONING` with a new list of partitions.